src/nxdoc/getting-started/tracks/package-publish-deploy-hotreload.md

```diff
@@ -0,0 +1,160 @@
+---
+title: 4 - Package \ Publish \ Deploy \ Hotreload
+review:
+    comment: ''
```

Comment:
The indentation in this file seems to be 4 spaces rather than 2 like the other files. Was there an issue that required this?

Reply:
There was no issue, just a mistake